app/Place/PlaceEditingServiceProvider.php

```diff
@@ -38,6 +38,15 @@ function ($app) {
                 );
             }
         );
+
+        $app[LocationMarkedAsDuplicateProcessManager::class] = $app->share(
```

Comment:
Only suggestion I'd have is that this might be more related to events so we might want to put it in the `EventEditingServiceProvider` instead, but it's a grey area so I'm fine with either one.

Reply:
I'll move it tomorrow morning ;)